src/main/java/uk/gov/justice/hmpps/offenderevents/services/HMPPSDomainEventsEmitter.kt

```diff
@@ -46,13 +45,13 @@ class HMPPSDomainEventsEmitter(
   fun convertAndSendWhenSignificant(event: OffenderEvent) {
     val hmppsEvents =
       if (event.caseNoteId != null) {
-        toCaseNotePublished(event).stream().toList()
+        listOfNotNull(toCaseNotePublished(event))
```

Comment:
hadn't seen `listOfNotNull` but is quite nice and will convert `null` to `emptyList` otherwise will be singleton list